ui/imports/shared/popups/ImportCommunityPopup.qml

```diff
@@ -49,85 +60,136 @@ StatusDialog {
+                color: Theme.palette.directColor1
+                font.pixelSize: 15
+            }
+
+            StatusTextArea {
+                id: keyInput
+                Layout.fillWidth: true
+                implicitHeight: 108
+                placeholderText: "0x0..."
+                wrapMode: TextEdit.WrapAtWordBoundaryOrAnywhere
+                onTextChanged: d.importErrorMessage = ""
+            }
+            RowLayout {
+                Layout.fillWidth: true
+                Layout.minimumHeight: 46
+                Layout.maximumHeight: 46
+                StatusChatInfoButton {
+                    visible: d.communityFound
+                    title: visible ? d.communityDetails.name : ""
+                    subTitle: visible ? qsTr("%n member(s)", "", d.communityDetails.nbMembers) : ""
+                    asset.name: visible ? d.communityDetails.image : ""
+                    asset.isImage: (asset.name !== "")
+                    asset.color: visible ? d.communityDetails.color : ""
+                }
+                Item { Layout.fillWidth: true }
+                StatusBaseText {
+                    id: detectionLabel
+                    Layout.alignment: Qt.AlignRight
+                    font.pixelSize: 13
+                    visible: keyInput.text.trim() !== ""
+                    text: {
+                        if (d.errorMessage !== "") {
+                            return d.errorMessage
+                        }
+                        if (d.isPrivateKey) {
+                            return qsTr("Private key detected")
+                        }
+                        if (d.isPublicKey) {
+                            return qsTr("Public key detected")
+                        }
                     }
-                    if (d.isPublicKey) {
-                        return qsTr("Public key detected")
+                    color: d.errorMessage === "" ? Theme.palette.successColor1 : Theme.palette.dangerColor1
+                }
+            }
+            ColumnLayout {
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+                visible: (d.communityFound && d.isPrivateKey)
+                Layout.topMargin: 12
+                spacing: Style.current.padding
+                Rectangle   {
+                    Layout.fillWidth: true;
+                    implicitHeight: 80
+                    radius: Style.current.radius
+                    color: "transparent"
+                    border.color: Theme.palette.warningColor1
+                    RowLayout {
+                        anchors.fill: parent
+                        anchors.leftMargin: Style.current.padding
+                        anchors.rightMargin: Style.current.padding
+                        spacing: 6
+                        StatusIcon {
+                            Layout.alignment: Qt.AlignTop
+                            Layout.topMargin: 4
+                            icon: "caution"
```

Comment:
The `caution` icon is not aligned.

<img width="631" alt="Screenshot 2023-08-03 at 22 24 36" src="https://github.com/status-im/status-desktop/assets/47811206/0158d3e4-3b35-48ec-a853-f6df1a3bdf50">